azkv/keysource.go

```diff
@@ -120,7 +120,15 @@ func (t TokenCredential) ApplyToMasterKey(key *MasterKey) {
 
 // Encrypt takes a SOPS data key, encrypts it with Azure Key Vault, and stores
 // the result in the EncryptedKey field.
+//
+// Deprecated: use EncryptContext instead.
```

Comment:
Deprecating this is quite aggressive. I would simply hint that a context accepting alternative is available.

Reply:
Fixed, thanks!